README.md

```diff
@@ -2,6 +2,14 @@
 
 The orders resource is a collection of order items where each item represents a product, its quantity, and its price. Since this is really a collection of order items, we will implement a subordinate REST API to place order items into the order collection (e.g., /orders/{id}/ items). We will also associate the order with a customer through its customer id.
 
+Order API
+ 
+* /orders : This is the API to create an empty order. This will create an order for customer. The order has following fields:
+  customer_id, address_line1, address_line2, city, state, zip_code.
+
+* /orders/{id}/items : This is the API to create an item inside an order. Items have following field:
```

Comment:
I don't see the corresponding route for `/orders/{id}/items. Do you have the code ready to commit?

Reply:
No, Item model is not ready yet however I have posted the design in Readme so that there should not be any blocker in progress for the team and you can refer the details from there . I would be using exactly same keywords.